build.gradle

```diff
@@ -145,7 +151,11 @@ project(':iceberg-api') {
   }
 }
 
-project(':iceberg-common') {}
+project(':iceberg-common') {
```

Comment:
Do you not also need to do this for iceberg-arrow? Or does it get the `relocate-external-dependencies` transitively?

Reply:
yeah, it can get them transitively here

Reply:
OK, I don't like the fact that this is how it works, I prefer declaring what I know I need but let's not change it for now.